src/peft/utils/other.py

```diff
@@ -21,7 +21,7 @@
 import accelerate
 import torch
 from accelerate.hooks import add_hook_to_module, remove_hook_from_module
-from accelerate.utils import is_npu_available, is_xpu_available
+from accelerate.utils import is_npu_available, is_xpu_available, is_mlu_available
```

Comment:
accelerate >= v0.29.0 is OK. accelerate has been released. Should I change setup.py with accelerate>=0.29.0 ?

Reply:
Let's rather implement this in a way that would not break with older accelerate versions.

Reply:
Yes you can do a version check of `accelerate` here and initilize `is_mlu_available = None` and replace it with the import in case the version is greater than 0.29.0